apps/src/studio/skins.js

```diff
@@ -931,8 +931,47 @@ function loadHoc2015(skin, assetUrl) {
     'applause'
   ];
 
+  skin.soundGroups = {
+    'R2-D2sound': {
+      randomValue: 'R2-D2random',
+      minSuffix: 1,
+      maxSuffix: 9
```

Comment:
I have an idea for how to clean this up a little, but I'm going to do it later since there are some higher priority items I'd like to complete first.